docs/customization/custom-scripts.md

```diff
@@ -56,6 +56,15 @@ class AnotherCustomScript(Script):
 script_order = (MyCustomScript, AnotherCustomScript)
 ```
 
+### The run() method
+
+The run() method is the entrypoint for the script, and it runs in the context of Netbox's own execution environment. This means from here, everything inside Netbox itself is accesible. The [Netbox Shell](../administration/netbox-shell.md) is a good resource to keep in hand, since it allows to see the objects in Netbox in the same way the run() method of the script does.
```

Comment:
NetBox is always written as either "NetBox" (proper usage) or "netbox" (referencing code/resources); never "Netbox."

Reply:
Done. Sorry for the mistake